docs/installation.rst

```diff
@@ -5,13 +5,13 @@ Installation
 Requirements
 ------------
 Requires:
- * `numpy`
- * `scipy`
- * `aipy`
- * `hera_cal` (which requires `h5py`)
- * `pyuvdata`
+ * ``numpy``
+ * ``scipy``
+ * ``aipy``
+ * ``hera_cal`` (which requires ``h5py``)
```

Comment:
It looks like the only place `hera_cal` is used is in `vis.py`, and it's just importing `utils.split_bl` from `hera_cal` inside the `sim_red_data` function. I don't think this PR is the place to deal with this, but I think we may want to think a bit about whether it's sensible to keep `hera_cal` as a requirement given its extremely limited use.

Reply:
I've made a new extra requires for `cal` which installs `hera_cal`, and otherwise removed it from installation instructions (it wasn't being installed automatically anyway).